wdns/str_to_rdata_ubuf.c

```diff
@@ -197,31 +197,65 @@ str_to_svcparam(ubuf *u, char *keyval)
 		} while ((tok = strtok_r(NULL, ",", &endp)) != NULL);
 		break;
 
-	case spr_alpn:
+	case spr_alpn: {
 		/*
 		 * The wire format value for "alpn" consists of at least one
 		 * "alpn-id" prefixed by its length as a single octet, and
 		 * these length-value pairs are concatenated to form the
 		 * SvcParamValue. These pairs MUST exactly fill the
 		 * SvcParamValue; otherwise, the SvcParamValue is malformed.
 		 */
-		tok = strtok_r(NULL, ",", &endp);
+		char *src;
+		int len;
+		ubuf *v;
+		uint8_t c, bytes;
+
+		tok = strtok_r(NULL, " \t\r\n", &endp);
```

Comment:
See comment on wdns/rdata_to_ubuf.c

This also affects the `default:` case of this switch statement, handling `keyNNN` values